workers/unity/Packages/io.improbable.gdk.tools/GenerateCode.cs

```diff
@@ -476,5 +450,94 @@ private static IEnumerable<string> FindDirInPackages(string directory)
 
             return packagePaths.Union(cachedPackagePaths).Distinct();
         }
+
+        internal static void GenerateCodegenRunConfigs(GdkToolsConfiguration toolsConfig = null)
```

Comment:
Why does this have a default argument of `null`?

Reply:
remnant of an old idea 😅